app/src/main/java/mozilla/lockbox/autofill/FillResponseBuilder.kt

```diff
@@ -26,8 +26,8 @@ class FillResponseBuilder(
         val responseBuilder = FillResponse.Builder()
 
         val authPresentation = RemoteViews(context.packageName, R.layout.autofill_item).apply {
-            val callToAction = context.resources.getString(R.string.autofill_authenticate_cta)
-            setTextViewText(R.id.presentationText, callToAction)
+            val searchFirefoxLockbox = context.resources.getString(R.string.autofill_authenticate_cta)
+            setTextViewText(R.id.searchLockboxText, searchFirefoxLockbox)
```

Comment:
This `authPresentation` has a variable called `searchFirefoxLockbox`. I think I'd stick with the identifier names that are already there.